catalog/app/containers/NavBar/NavBar.tsx

```diff
@@ -19,30 +19,39 @@ import bg from './bg.png'
 
 import Controls from './Controls'
 
-const LogoLink = (props) => {
+function LogoLink(props: M.BoxProps) {
   const { urls } = NamedRoutes.use()
   return (
+    // @ts-ignore
     <M.Box component={Link} mr={2} to={urls.home()} {...props}>
       <Logo responsive />
     </M.Box>
   )
 }
 
-const Item = React.forwardRef((props, ref) => (
-  <M.MenuItem
-    // eslint-disable-next-line no-nested-ternary
-    component={props.to ? Link : props.href ? 'a' : undefined}
-    ref={ref}
-    {...props}
-  />
-))
+interface ItemProps {
+  to?: string
```

Comment:
Actually, `to` accepts objects too. And we can set two types union (one OR another) to avoid passing `to` and `href` the same time:
```ts
type BrowserLinkProps = { href: string } & MenuItemProps;
type RouterLinkProps = LinkProps & MenuItemProps;
type ItemProps = BrowserLinkProps | RouterLinkProps;
```

Reply:
makes sense, but doesnt typecheck. we'll revisit it someday soon